docs: Update quality settings to match the UI (#8777)
heartexlabs/label-studio: docs/source/guide/project_settings_lse.md

Reviewer: Let me confirm with Alec if we want to bring this back, because it's currently not possible to set this to zero.

Reviewer: Circling back, Alec confirmed we need to remove this last sentence:

> If left blank, there is no limit to additional annotators.

We will not bring this functionality back.

| Field          | Description    |
| ------------- | ------------ |
| **Allow empty annotations** | This determines whether annotators can submit a task without making any annotations on it. If enabled, annotators can submit a task even if they haven't added any labels or regions, resulting in an empty annotation. |

> | **Show the Data Manager to annotators** | When disabled, annotators can only enter the label stream. When enabled, annotators can access the Data Manager. The tasks that are available to them depend on the how task are assigned: <ul><li>Automatic task assignment: Annotators can only see tasks that they have already completed or have created a draft for.</li><li>Manual task assignment: Annotators can only see the tasks that they have been assigned.</li></ul>Note that some information is still hidden from annotators and they can only view a subset of the Data Manager columns. For example, they cannot see columns such as Annotators, Agreement, Reviewers, and more. |

Reviewer: ```suggestion
| **Show Data Manager to annotators** | When disabled, annotators can only enter the label stream. When enabled, annotators can access the Data Manager. The tasks that are available to them depend on the how task are assigned: <ul><li>Automatic task assignment: Annotators can only see tasks that they have already completed or have created a draft for.</li><li>Manual task assignment: Annotators can only see the tasks that they have been assigned.</li></ul>Note that some information is still hidden from annotators and they can only view a subset of the Data Manager columns. For example, they cannot see columns such as Annotators, Agreement, Reviewers, and more. |
```

!!! note
    Overlap settings only apply when the project is using Automatic distribution mode. If you are using Manual distribution mode, all tasks must be manually assigned - meaning that you are also manually determining overlap.  

> By default, each task only needs to be annotated by one annotator. If you want multiple annotators to be able to annotate tasks, increase the **Annotations per task**.

Reviewer: ```suggestion
By default, each task only needs to be annotated by one annotator. If you want multiple annotators to annotate the same tasks, increase the **Annotations per task**.
```

<table>
<thead>
    <tr>
      <th>Field</th>
      <th>Description</th>
    </tr>
</thead>
<tr>
<td>

**Annotations per task**
</td>
<td>

The number of annotations you want to allow per task. 

Note that in certain situations, this may be exceeded. For example, if there are long-standing drafts within a project or you have a very low [task reservation](#task-lock) time. 

</td>
</tr>
<tr>
<td>

**Annotations per task coverage**
</td>
<td>

Only available if **Annotations per task** is ≥ 2. 

> This setting controls how the overlap should be enforced. 

Reviewer: ```suggestion
This setting controls the percentage of the project tasks for which the overlap is enforced. 
```

Use this option to determine what types of tasks annotators will see first. 

> * **Ongoing** - Annotators are presented with tasks in the order that is configured under [**Task Ordering**](#task-ordering ). 

Reviewer: ```suggestion
* **Ongoing** - Annotators are presented with tasks in the order that is configured under [**Task Ordering Method**](#task-ordering ). 
```

<tr>
<td>

> **Low agreement strategy**

Reviewer: I believe we are missing the header of the section, which was added very recently.

"Low Agreement Resolution", which was previously Low Agreement Strategy.

The "Assign additional annotator" is a toggle under it.

Author: Yes, I left that out because it was just a header and wouldn't work in the table.
